app/views/RoomView/index.tsx

```diff
@@ -1260,13 +1253,6 @@ class RoomView extends React.Component<IRoomViewProps, IRoomViewState> {
 				this.goRoomActionsView();
 			} else if (handleCommandSearchMessages(event)) {
 				this.goRoomActionsView('SearchMessagesView');
-			} else if (handleCommandReplyLatest(event)) {
```

Comment:
Key commands are crashing right now.
I'm going to remove in a future PR and plan a new implementation later.